src/classes/Extend/ThemesDenylist.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Extends the GDPR functionality.
+ */
+
+namespace Niteo\WooCart\Defaults\Extend;
+
+trait ThemesDenylist {
+
+	/**
+	 * Adds notice in the admin panel for the denylisted theme.
+	 *
+	 * @return void
+	 */
+	public function add_denylist_theme_notice() : void {
+		$current_theme = \wp_get_theme();
+
+		$message = sprintf(
+			esc_html__( '%1$s theme has been denylisted on WooCart. Kindly switch to a different theme or %2$scontact support%3$s.', 'woocart-defaults' ),
```

Comment:
```suggestion
			esc_html__( '%1$s theme has been denylisted on WooCart because of poor performance. We recommend switching to a different theme.', 'woocart-defaults' ),
```